docs/source/geo_def.rst

```diff
@@ -68,7 +68,7 @@ area defintions. The function **get_area_def** can construct an area definition
 based on area extent and a proj4-string or a list of proj4 arguments.
 
 .. doctest::
-	
+
  >>> from pyresample import utils
  >>> area_id = 'ease_sh'
  >>> area_name = 'Antarctic EASE grid'
```

Comment:
I think this should be named "description" instead of "area_name" so it would match naming of the corresponding YAML field